frontend/public/components/utils/request-size-input.tsx

```diff
@@ -33,6 +33,7 @@ export class RequestSizeInput extends React.Component<RequestSizeInputProps> {
             aria-describedby={describedBy}
             name={inputName}
             required={this.props.required}
+            value={this.props.defaultRequestSizeValue}
           />
           <Dropdown
             title={this.props.defaultRequestSizeUnit}
```

Comment:
If we only allow one value for selection, why not disable this dropdown field for now?

Reply:
@shirimordechay make sense.